spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -384,3 +384,81 @@ dataflow:>task execution list
 ```
 dataflow:>task destroy --name task1
 ```
+
+== Application Configuration
+
+This section covers how you can customize the deployment of your applications. You can use a number of deployer properties to influence settings for the applications that are deployed.
+
+See https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/blob/master/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java[KubernetesDeployerProperties] for more of the supported options.
+
+If you would like to override the default values for all apps that you deploy then you should modify the <<index.html#_spring_cloud_deployer_for_kubernetes_properties, Spring Cloud Deployer for Kubernetes Properties>> for the server.
+
+=== Memory and CPU Settings
+
+The Spring Cloud Deployer that is being used has certain defaults for the memory and CPU setting that are used for deployed apps. With the defaults the container Limits and Requests are set to the following:
```

Comment:
> The Spring Cloud Deployer that is being used has certain defaults for the memory and CPU setting that are used for deployed apps. With the defaults the container Limits and Requests are set to the following:

"The apps are deployed by default with the following "Limits" and "Requests" settings."